services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/bsm/OdeBsmJsonRepository.java

```diff
@@ -0,0 +1,11 @@
+package us.dot.its.jpo.ode.api.accessors.bsm;
+
+import java.util.List;
+
+import us.dot.its.jpo.ode.api.models.DataLoader;
+import us.dot.its.jpo.ode.model.OdeBsmData;
+
+public interface OdeBsmJsonRepository extends DataLoader<OdeBsmData>{
+    List<OdeBsmData> findOdeBsmDataGeo(String originIp, String vehicleId, Long startTime, Long endTime, Double longitude, Double latitude, Double distance);
```

Comment:
suggestion: Are we finding OdeBsmData objects within a given radius from a point? I would assume so from the parameters but it's not immediately clear. Can we get some XML or javadoc comments in here to help clarify/document?

Reply:
Added comments, and re-worked this to use org.geotools